sympy/stats/joint_rv.py

```diff
@@ -226,24 +226,27 @@ def _sample_numpy(cls, dist, size, seed):
         return samples.reshape(size + sample_shape[dist.__class__.__name__](dist))
 
 class SampleJointPymc:
-    """Returns the sample from pymc3 of the given distribution"""
+    """Returns the sample from pymc of the given distribution"""
 
     def __new__(cls, dist, size, seed=None):
-        return cls._sample_pymc3(dist, size, seed)
+        return cls._sample_pymc(dist, size, seed)
 
     @classmethod
-    def _sample_pymc3(cls, dist, size, seed):
-        """Sample from PyMC3."""
-
-        import pymc3
-        pymc3_rv_map = {
+    def _sample_pymc(cls, dist, size, seed):
+        """Sample from PyMC."""
+
+        try:
+            import pymc
+        except ImportError:
+            import pymc3 as pymc
```

Comment:
Maybe add a comment here noting that we should remove this whenever pymc3 gets sufficiently superceeded by pymc 4 (or whenever supporting both becomes a burden).